homeassistant/components/camera/local_file_camera.py

```diff
@@ -0,0 +1,54 @@
+"""Camera that loads a picture from a local file."""
+
+import logging
+import os
+
+from homeassistant.components.camera import Camera
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Camera."""
+    if config.get("file_path") is None:
+        _LOGGER.error("Missing required variable: file_path")
+        return False
+
+    setup_config = (
+        {
+            "name": config.get("name", "Local File Camera"),
+            "file_path": config.get("file_path",
+                                    os.path.join(os.path.dirname(__file__),
+                                                 'image.jpg'))
```

Comment:
would it make more sense for the name to be the file path, if no name is provide, rather then just " Local Camera File"